app/views/jobs/_index.html.haml

```diff
@@ -4,10 +4,10 @@
       %h1 Jobs
     #list-item.flex
     - @jobs.each do |job|
-      #form-name
-        %p= job.name
-      #form-button
-        %button.button Remove
+      #form-grid  
```

Comment:
I see no form here. Why use an I’d like this?

Reply:
Legacy. See other comments